docs/INSTALLATION.md

```diff
@@ -0,0 +1,265 @@
+
+```sh
+docker-compose -f docker-compose.assets.yml up -d
+```
+
+Once the services have started, you can check their status by running:
+
+```sh
+docker ps
+```
+
+The `ps` command will list containers along with their `CONTAINER_ID`. Then, to check logs of an individual container, run:
+
+```sh
+docker logs <CONTAINER_ID>
+```
+
+To stop the Docker runtime run:
+
+```sh
+docker-compose -f docker-compose.ove.yml down
+```
+
+To clean-up the Docker runtime first stop any active instances and then run:
+
+```sh
+docker system prune
+docker volume prune
+```
+
+## Installation from source code
+
+All OVE projects use a build system based on [Lerna](https://lernajs.io/). [**OVE Asset Services**](https://github.com/ove/ove-asset-services) are based on [.NET Core](https://github.com/dotnet/core). All other OVE projects are based on [Node.js](https://nodejs.org/en/), compiled with [Babel](http://babeljs.io/), and deployed on a [PM2](http://pm2.keymetrics.io/) runtime.
+
+### Prerequisites
+
+* [git](https://git-scm.com/downloads)
+* [Node.js](https://nodejs.org/en/) (v8.0+)
+* [NPM](https://www.npmjs.com/)
+* [NPX](https://www.npmjs.com/package/npx) (install with the command: `npm install -global npx`)
+* [PM2](http://pm2.keymetrics.io/) (install with the command: `npm install -global pm2`)
+* [Lerna](https://lernajs.io/) (install with the command: `npm install -global lerna`)
+
+Compiling source code for the Docker runtime also requires:
+
+* [Docker](https://www.docker.com/get-started)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+Building [**OVE Asset Services**](https://github.com/ove/ove-asset-services) also requires:
+
+* [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+* [NetVips](https://github.com/kleisauke/net-vips)
+* [MariaDB](https://mariadb.org/)
+
+The [SVG App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-svg) requires:
+
+* [Tuoris](https://github.com/fvictor/tuoris) (installation instructions available on GitHub repository)
+
+### Downloading source code
+
+All OVE projects can be downloaded from their GitHub repositories:
+
+* OVE Core: [master](https://github.com/ove/ove) | [releases](https://github.com/ove/ove/releases)
+* OVE Apps: [master](https://github.com/ove/ove-apps) | [releases](https://github.com/ove/ove-apps/releases)
+* OVE Services: [master](https://github.com/ove/ove-services) | [releases](https://github.com/ove/ove-services/releases)
+* OVE Asset Services: [master](https://github.com/ove/ove-asset-services) | [releases](https://github.com/ove/ove-asset-services/releases)
+
+The `master` branch of each repository contains the latest code, and can also be cloned if you intend to contribute code or fix issues:
+
+```sh
+git clone https://github.com/ove/ove
+```
+
+Once the source code has been downloaded OVE can be installed either onto a PM2 runtime or a Docker runtime. The two approaches are explained below.
```

Comment:
This is not "either/or". Docker is not a runtime here, it is a container for pm2